src/app/exercise/services/deps-order.service.ts

```diff
@@ -0,0 +1,79 @@
+import { Injectable } from '@angular/core';
+import { FileConfig } from '../interfaces/file-config';
+
+import * as ts from 'typescript';
+import { simpleVisitor } from './visitor';
+
+
+@Injectable()
+export class DepsService {
+
+  /**
+   * Takes a file path, and another path relative to the first one.
+   * Returns the fulle
+   *
+   * @param path
+   * @param file
+   * @returns {string}
+   */
+  static normalizePathRelativeToFile(path, file) {
+    // TODO: Simplify
+    let fullPath = '/' + path.substr(0, path.lastIndexOf('/')) + '/' + file;
+    while (fullPath.match(/\/[^/]*\/\.\./)) {
+      fullPath = fullPath.replace(/\/[^/]*\/\.\./, '');
+    }
+    fullPath = fullPath.replace(/\/.\//, '/');
+    return fullPath.replace(/^\//, '').replace(/^\//, '');
+  }
+
+  static isLocalDep(string) {
```

Comment:
There are no tests for this method specifically, is this supposed to be a private method?